### ISSUE TITLE
Small Audio Suggestion

### ISSUE THREAD

**Issue by Degamisu** [valadaptive/ntsc-rs#30]
When I digitized my tape, I noticed that there was a high pitch ringing noise. I slowed it down and it sounded like a sine wave. Not sure what it was, but it sounded like a CRT screen. Can you add this? Thanks!

**Comment by Degamisu** [comment]
Yes. I notice this too, It happens to all my digitized tapes. While i couldn't find anything on this noise, but i do have a theory. This could be because the audio bandwidth is so high it bleeds into the video band, adding the high pitched sine wave. This would definetly be useful for realism.

**Comment by valadaptive** [comment]
...Did you just thumbs-up your own issue?

**Comment by Degamisu** [comment]
LMAO i was half awake when i saw this and i thought it was someone else so i replied to myself 😆
